src/services/forest-server-requester.js

```diff
@@ -1,11 +1,12 @@
 const P = require('bluebird');
 const superagent = require('superagent');
 const VError = require('verror');
-const ServiceUrlGetter = require('./service-url-getter');
 const errorMessages = require('../utils/error-messages');
+const context = require('../context');
```

Comment:
[CLEANING] You might want to do this instead: 
`const { inject } = require('../context');`